src/gettingstarted/own-code/connect-services.md

```diff
@@ -0,0 +1,157 @@
+# Import your own code
+
+## Connect to Services
+
+> **Note:** If your application does not require access to services and you left `.platform/services.yaml` blank, feel free to proceed to the next step.
+
+At this point you have configured your application's services as well as its build and deploy process in your `.platform/services.yaml` and `.platform.app.yaml` files. As an example, in your `.platform.app.yaml` file you may have defined a relationship called `database`:
+
+```yaml
+relationships:
+    database: "mysqldb:mysql"
+```
+
+which was configured in `.platform/services.yaml` with
+
+```yaml
+mysqldb:
+    type: mysql:10.2
+    disk: 1024
+```
+
+In order to connect to this service and use it in your application, Platform.sh exposes its credentials in the application container within a base64-encoded JSON `PLATFORM_RELATIONSHIPS` environment variable.
+
+To access this variable you can install a Platform.sh configuration reader library
+
+{% codetabs name="PHP", type="php" -%}
+composer install platformsh/config-reader
+{%- language name="Python", type="py" -%}
+pip install platformshconfig
+{%- language name="Node.js", type="js" -%}
+npm install platformsh-config --save
+{%- language name="Java", type="java" -%}
+// Installing with Maven
+ <dependency>
+     <groupId>sh.platform</groupId>
+     <artifactId>config</artifactId>
+     <version>0.0.1-SNAPSHOT</version>
+ </dependency>
+
+// Installing with Gradle
+compile group: 'sh.platform', name: 'config', version: '0.0.1-SNAPSHOT'
+{%- language name="Go", type="go" -%}
+// In your go.mod file
+github.com/platformsh/config-reader-go/v2 v2.2.2
+{%- endcodetabs %}
+
+
+and access the credentials of `database`
+
+{% codetabs name="PHP", type="php" -%}
+use Platformsh\ConfigReader\Config;
+
+$config = new Config();
+$credentials = $config->credentials('database');
+{%- language name="Python", type="py" -%}
+from platformshconfig import Config
+
+config = Config()
+credentials = config.credentials('database')
+{%- language name="Node.js", type="js" -%}
+const config = require("platformsh-config").config();
+const credentials = config.credentials('database');
+{%- language name="Java", type="java" -%}
+import Config;
+
+Config config = new Config();
+Credential cred = config.getCredential('database')
+{%- language name="Go", type="go" -%}
+package main
+
+import (
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
+	sqldsn "github.com/platformsh/config-reader-go/v2/sqldsn"
+	psh "github.com/platformsh/config-reader-go/v2"
+	"net/http"
+)
+
+func main() {
+
+	// Creating a psh.RuntimeConfig struct
+	config, err := psh.NewRuntimeConfig()
+	if err != nil {
+		panic("Not in a Platform.sh Environment.")
+	}
+
+	// Accessing the database relationship Credentials struct
+	credentials, err := config.Credentials("database")
+	if err != nil {
+		panic(err)
+	}
+
+  ...
+
+}
+{%- endcodetabs %}
```

Comment:
Poor Go and it's absurdly long code example...  Can that be made shorter?

Reply:
Cut most of it out.